server/game/cards/01-DTR/MakeTheSmartChoice.js

```diff
@@ -0,0 +1,49 @@
+const ActionCard = require('../../actioncard.js');
+
+class MakeTheSmartChoice extends ActionCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Make the Smart Choice',
+            playType: ['shootout'],
+            target: {
+                activePromptTitle: 'Choose a dude',
+                choosingPlayer: 'current',
+                cardCondition: { location: 'play area', controller: 'any', participating: true },
+                cardType: ['dude']
+            },
+            handler: context => {
+                // "Reduce a dude's bullets by their influence."
+                const influence = context.target.influence;
+                if(influence) {
+                    this.applyAbilityEffect(context.ability, ability => ({
+                        match: context.target,
+                        effect: ability.effects.modifyBullets(-1 * influence)
+                    }));
+                    this.game.addMessage(
+                        '{0} uses {1} to reduce {2}\'s bullets by their influence ({3})',
+                        context.target.controller,
```

Comment:
Incorrect player using the ability.
```suggestion
                        context.player,
```